lte/gateway/dev_tools.py

```diff
@@ -67,6 +68,42 @@ def register_vm():
     _register_agw(LTE_NETWORK_TYPE)
 
 
+def register_vm_remote():
+    """
+    Register local VM gateway with remote controller.
+
+    Example usage:
+    fab --set certs_dir=~/certs,network_id=test,url=https://api.stable.magmaeng.org \
+        -f dev_tools.py register_vm_remote
+    """
+    certs_dir = env.get('certs_dir')
+    url = env.get('url')
+    network_id = env.get('network_id')
+    if None in [certs_dir, url, network_id]:
+        print()
+        print(f'============================================================')
+        print(f'Must --set the following variables: certs_dir,url,network_id')
+        print(f'============================================================')
+        return
+
+    admin_cert = types.ClientCert(
+        cert=os.path.expanduser(f'{certs_dir}/admin_operator.pem'),
+        key=os.path.expanduser(f'{certs_dir}/admin_operator.key.pem'),
+    ),
```

Comment:
you are creating the tuple adding that `,`

Reply:
I literally spent 30 minutes trying to debug that LOL. Thanks for the catch!